### ISSUE TITLE
[BUG] Long tweets by Twitter Blue users are not shown completely

### ISSUE THREAD

**Issue by mecuricat** [jonjomckay/fritter#644]
**Describe the bug**
Twitter Blue users can write longer tweets but Fritter fails to show complete tweet.

**To Reproduce**
1. Find any [long tweet](https://twitter.com/MishaalRahman/status/1629319713107894274).

**Screenshots**
#### Tweet screenshot from Fritter
<img src="https://user-images.githubusercontent.com/89805719/221376712-0fac0685-b3ab-40bc-b43f-5b89f7697f11.jpg" width="200px"/>

#### Tweet screenshot from browser
<img src="https://user-images.githubusercontent.com/89805719/221376734-1a01b892-0d5a-4262-ac93-2458bd091c3d.jpg" width="200px"/>

**Device**
Redmi Note 10 Pro, Android 12, English, and Fritter v3.0.0-beta5.

**Comment by SudipRajbanshi** [comment]
Im also facing the same issue

**Comment by Gotchfutchian** [comment]
Also putting my obligatory 'same here'

**Comment by mtaki14** [comment]
This is my main issue with the app at the moment, othewise it's pretty much perfect for my needs. It's impossible to read long tweets

**Comment by ballo** [comment]
Thankfully Elon has made the Twitter web site more accessible to non-users so the current work-around is to just stop using fritter

**Comment by nannadroid** [comment]
Yes, same issue. Is this already released? If yes, in which version?
